.gitignore

```diff
@@ -0,0 +1,17 @@
+# See https://help.github.com/articles/ignoring-files for more about ignoring files.
+#
+# If you find yourself ignoring temporary files generated by your text editor
+# or operating system, you probably want to add a global ignore instead:
+#   git config --global core.excludesfile '~/.gitignore_global'
+
```

Comment:
@CheezItMan thanks for the feedback!  I added this in a separate branch that was just merged.  I don't think I shared the keys online - at least I can't see them anywhere... for the future, is there a way to delete this stuff from git if we do accidentally add them?  Thanks again!